src/Home/index.spec.tsx

```diff
@@ -16,9 +15,6 @@ import { ConfigurationContext } from "../ConfigurationContext"
 
 jest.mock("@react-navigation/native")
 
-jest.mock("react-native-safe-area-context")
-;(useSafeAreaInsets as jest.Mock).mockReturnValue({ insets: { bottom: 0 } })
```

Comment:
nice!! mocks that are not relevant to most of the tests go great under the `mocks` folder 👏

Reply:
😋